### PR TITLE
editors: improve styles

### PR DESCRIPTION
**Pull request by sebastiendeleze**
Improves styles of editors by customizing the card behavior.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

File: projects/sonar/src/styles.scss [hunk 1]
```diff
@@ -62,5 +62,23 @@ pre {
 }
 
 .card.wrapper {
-  @extend .my-3
+  border-radius: 0;
```
Comment (zannkukai) [inline projects/sonar/src/styles.scss:65]: seems useless if border is none

File: projects/sonar/src/styles.scss [hunk 2]
```diff
@@ -62,5 +62,23 @@ pre {
...
+    padding-left: 0;
+  }
+
+  .card-body {
+    padding-left: 0;
+  }
+
+  .border-left {
+    padding-left: 1rem !important;
+    margin-bottom: 1rem !important;
```
Comment (zannkukai) [inline projects/sonar/src/styles.scss:81]: maybe use `map_get($spacers, 3)` instead of 1rem ?

File: projects/sonar/src/styles.scss [hunk 3]
```diff
@@ -62,5 +62,23 @@ pre {
...
+    border-left: 3px solid #dee2e6 !important
```
Comment (zannkukai) [inline projects/sonar/src/styles.scss:82]: is this color could be defined into `variables.scss` file.

Comment (sebastiendeleze) [reply projects/sonar/src/styles.scss:82]: I promise, if I use this color one more time, I will set a variable for it!

**Pull request merged**